packages/backend-lib/src/destinations/twilio.ts

```diff
@@ -46,3 +62,67 @@ export async function sendSms({
     return err(error);
   }
 }
+
+export async function submitTwilioEvents({
+  workspaceId,
+  TwilioEvent,
+}: {
+  workspaceId: string;
+  TwilioEvent: TwilioInboundSchema;
+  subscriptionGroupId: string | undefined;
+}): Promise<ResultAsync<void, Error>> {
+  const messageBody = TwilioEvent.Body?.toLowerCase();
+
+  let subscriptionStatus = null;
+  if (messageBody?.includes("stop")) {
+    subscriptionStatus = SubscriptionGroupType.OptOut;
+  } else if (messageBody?.includes("start")) {
+    subscriptionStatus = SubscriptionGroupType.OptIn;
```

Comment:
So, the `SubscriptionGroupType` is used to define whether users are assumed to be in the group before they opt-out, or assumed to be out of the group until they opt-in. You're probably looking for the `UserSubscriptionAction` type.